extensions/amazon-search/package.json

```diff
@@ -11,7 +11,7 @@
   "license": "MIT",
   "commands": [
     {
-      "name": "index",
+      "name": "search-amazon",
```

Comment:
Can we keep the old name? hotkey, alias and ranking it bound to this identifier inside the extension 🙂

Reply:
Done.